plugin.video.catchuptvandmore/resources/lib/enum/LICENSE

```diff
@@ -0,0 +1,32 @@
+Copyright (c) 2013, Ethan Furman.
```

Comment:
I'm pretty sure that we don't like having Python modules embedded in addons.  The correct process is to create the needed module as an addon and then making it a required dependency.

Reply:
Nope, we don't. There needs to be a separate submission for this module. There were previous attempts, but no add-on using it. See https://github.com/xbmc/repo-scripts/pull/1334 for the latest. It was for Leia. So there should be new Python3 version submitted to the Matrix repo-scripts.